pkg/cliplugins/workload/plugin/syncer.yaml

```diff
@@ -39,6 +33,37 @@ rules:
   - "list"
   - "watch"
   - "delete"
+- apiGroups:
```

Comment:
We have to keep in mind that permissions given to the Syncer should be as minimal as possible.

Reply:
yes I'm well aware of this. thanks!

Reply:
So it seems like we need to add the names of the expected resources here, since we do *not* expect the syncer to manage every service/role/etc, yes?

Reply:
names are dynamically generated so we can't add them. The syncer manages all every services/roles/ in the syncer namespace only.